src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+        /// </feature>
+        /// <value>
+        /// The current state of the voice control service.
+        /// </value>
+        /// <pre>
+        /// The state must be ready.
+        /// </pre>
+        public static ServiceState ServiceState
+        {
+            get
+            {
+                ServiceState state;
+
+                ErrorCode error = VcMgrGetServiceState(out state);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "ServiceState Failed with error " + error);
+                    return ServiceState.Unavailable;
+                }
+
+                return state;
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the command format is supported.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="format">The command format</param>
+        /// <returns>The result status, true if supported</returns>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State must be Ready.
+        /// </pre>
+        public static bool IsCommandFormatSupported(CommandFormat format)
+        {
+            bool supported = false;
+            ErrorCode error = VcMgrIsCommandFormatSupported(format, out supported);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "IsCommandFormatSupported Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+            return supported;
+        }
+
+        /// <summary>
+        /// Sets system or exclusive commands.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="list">Command list</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void SetCommandList(VoiceCommandList list)
```

Comment:
What mean `Set` ?
What is a responsibility of this method?

Reply:
will fixed EventHandler.

Reply:
'setCommandList' means that it is to register a command list which includes several commands on vc-service.